lib/morpheus.gi

```diff
@@ -492,7 +492,8 @@ v, val, o, i, comb, best,actbase,action;
 	  end);
       fi;
 
-      if hom=fail and not IsAbelian(Socle(g)) and IsSimpleGroup(Socle(g)) then
+      if hom=fail and not IsAbelian(Socle(g))
+        and IsNonabelianSimpleGroup(Socle(g)) then
```

Comment:
Isn't the `IsAbelian` check redundant now? Alternatively, the switch from `IsSimpleGroup` to `IsNonabelianSimpleGroup` is redundant. (Not that it's important, the code is correct either way)

Reply:
Indeed it is. When I went through the changes I tried to not do anything nontrivial with the logic (so it could be understood easily).
Indeed here the `IsAbelian` can go now, will do.